### PR TITLE
Add thrift in the docker-sonic-mgmt

### PR DESCRIPTION
**Pull request by richardyu-ms**
#### Why I did it
In order to check the RPC connection from mgmt agent, there needs a thrift python package.

#### How I did it
Install the thrift 0.11 in docker-mgmt
#### How to verify it
local build docker-sonic-mgmt
#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [x] 202012
- [x] 202106

### CODE REVIEW

**Review by bingwang-ms — APPROVED** [review]

File: dockers/docker-sonic-mgmt/Dockerfile.j2 [hunk 1]
```diff
@@ -182,3 +182,6 @@ RUN pip install pycryptodome==3.9.8
 # Install allure-pytest library
 RUN pip install --upgrade setuptools \
     && pip install allure-pytest==2.8.22
+
+# Install thrift for checking rpc conection for sai test cases
+RUN pip install thrift==0.11.0
```
Comment (wangxin) [inline dockers/docker-sonic-mgmt/Dockerfile.j2:187]: Why net make it part of the `pip install` in line 36?

**Review by wangxin — APPROVED** [review]

**Comment by qiluo-msft** [comment]
This could not be cleanly cherry-pick to 202012. Please submit another PR.

**Comment by judyjoseph** [comment]
Please submit a PR for 202106 branch as well, as could not be cleanly cherry-picked

**Comment by richardyu-ms** [comment]
for nightly testing (this change intends to satisfy a dependency for nightly running tests), we don't need to deploy this, docker-sonic-mgmt, from branches 202012 and 202106. 
removing the tag for 202012 and 202106 backport requet

**Pull request merged**